lang/en.json

```diff
@@ -884,7 +887,8 @@
         "WarnDirectorBadSpend": "{name} attempted to spend a hero token while none were available.",
         "GainSurges": {
           "label": "Gain Surges",
-          "messageContent": "Spends a hero token to gain two surges."
+          "messageContent": "Spends a hero token to gain two surges.",
+          "dialogContent": "Spend one of {value} hero tokens to gain two surges?"
```

Comment:
Apparently hints in the form of a question are becoming a fashion faux pas and should be avoided.